src/Particular.ThroughputCollector.Persistence.RavenDb/Models/EndpointDocument.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Particular.ThroughputCollector.Persistence.RavenDb.Models;
+
+using Particular.ThroughputCollector.Contracts;
+
+class EndpointDocument(EndpointIdentifier id)
+{
+    public EndpointIdentifier EndpointId { get; } = id;
+
+    public string SanitizedName { get; set; } = string.Empty;
+
+    public IList<string> EndpointIndicators { get; } = new List<string>();
```

Comment:
simplify collection initialisation

Reply:
Yes, fixed it when build failed.  Not sure why I didn't get an error on my dev environment.